action.yml

```diff
@@ -8,8 +8,13 @@ inputs:
     description: 'Url you want to wait for'
     required: true
   timeout:
-    description: 'Custom timeout value (default: "10" (seconds))'
+    description: 'Duration (in seconds) to wait for a terminal deployment status'
     required: false
+    default: '10'
```

Comment:
See [Metadata syntax for GitHub Actions](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputsinput_iddefault) for context.